locust/html.py

```diff
@@ -19,7 +20,7 @@ def render_template(file, **kwargs):
 
 def get_html_report(environment, show_download_link=True):
     stats = environment.runner.stats
-
+    options = environment.parsed_options
```

Comment:
This isnt needed any more, right?

Reply:
Right, It's not needed. I'll delete this.